consensus/src/block_storage/sync_manager.rs

```diff
@@ -576,22 +629,23 @@ impl BlockRetriever {
                     .boxed(),
                 )
             }
-            let request = BlockRetrievalRequest::new_with_target_block_id(
+            let request = BlockRetrievalRequest::new_with_target_round(
```

Comment:
We cannot make this change until we have rolled out the new BlockRetievalRequestV2 handling logic in a release.  This should go in a future release after that one.